docs/modules/ROOT/pages/advanced-how-tos/releasing/create-release-plan-admission.adoc

```diff
@@ -67,5 +67,5 @@ kubectl apply -f ReleasePlanAdmission.yaml -n managed
 .*Verification*
 
 . In the {ProductName} UI, select the *Release services* > *Release plan admission* tab.
-. Review the RPA object that you just added. Using the Release plan admission tab, you can update or delete the selected Release plan object.
+. Review the RPA object that you just added. Using the Release plan admission tab, you can update or delete the selected Release plan admission object.
```

Comment:
I've never been to the RPA tab on the UI so don't know if one can update or delete the Release Plan from there. So, please correct me if I'm wrong.

Reply:
This seems reasonable. The tab is present for me but I don't belong to any tenants that have RPAs defined.